.eslintrc.json

```diff
@@ -1,10 +1,6 @@
 {
   "extends": "standard",
   "rules": {
-    "semi": [
-      "error",
-      "always"
-    ],
```

Comment:
Could we retain the original code style?

Reply:
Sure, I just removed it, so it's inline with other node-casbin packages.

Reply:
Reverted